### PR TITLE
Hide library followers for the default follow libs

### PR DESCRIPTION
**Pull request by ryanpbrewster**
@andrewconner

### CODE REVIEW

File: kifi-backend/modules/shoebox/app/com/keepit/commanders/LibraryInfoCommander.scala [hunk 1]
```diff
@@ -278,7 +278,7 @@ class LibraryInfoCommanderImpl @Inject() (
         val (collaboratorCount, followerCount, keepCount) = countsByLibraryId(libId)
```
Comment (andrewconner) [inline kifi-backend/modules/shoebox/app/com/keepit/commanders/LibraryInfoCommander.scala:280]: How about special casing this guy too?

Comment (ryanpbrewster) [inline kifi-backend/modules/shoebox/app/com/keepit/commanders/LibraryInfoCommander.scala:280]: The counts?

Comment (andrewconner) [inline kifi-backend/modules/shoebox/app/com/keepit/commanders/LibraryInfoCommander.scala:280]: Yeah, specifically `100911`

Comment (andrewconner) [inline kifi-backend/modules/shoebox/app/com/keepit/commanders/LibraryInfoCommander.scala:280]: ermmm `followerCount`

**Pull request merged**